app/controllers/concerns/alchemy/page_redirects.rb

```diff
@@ -0,0 +1,140 @@
+module Alchemy
+  module PageRedirects
+    extend ActiveSupport::Concern
+
+    included do
+      before_action :enforce_no_locale, only: [:index, :show],
+        if: :locale_prefix_not_allowed?
```

Comment:
Style/AlignHash: Align the elements of a hash literal if they span more than one line.